.github/workflows/m2k-func.yaml

```diff
@@ -3,10 +3,10 @@ name: Build and push move2kube Knative function container image
 on:
   workflow_call:
     inputs:
-      push_pr:
+      it_mode:
         required: false
         type: boolean
-        default: true
+        default: false
```

Comment:
are those changes and similar ones related to the PR or is this a rebase issue? I see there are 24 commits too